website/content/en/status/report-2023-04-2023-06/linuxulator.adoc

```diff
@@ -8,12 +8,12 @@ Contact: Dmitry Chagin <dchagin@FreeBSD.org>
 
 The goal of this project is to improve FreeBSD's ability to execute unmodified man:linux[4] binaries.
 
-As of gitref:cbbac5609115[repository=src], preserving of an fpu xsave state across signal delivery on amd64 is implemented.
+As of gitref:cbbac5609115[repository=src], preserving an fpu xsave state across signal delivery on amd64 is implemented.
 That is made possible to run modern golang with preemptive scheduler on.
```

Comment:
Grammar? I can not quite make sense of this sentence.

Reply:
I think what's meant is "That makes it possible to run..."

Reply:
@lemul: Could you please clarify what you meant with this sentence? Thanks.

Reply:
Resolved with [`068e91c` (#191)](https://github.com/freebsd/freebsd-doc/pull/191/commits/068e91cfa0a446f63caaf5faae1b1b45ac124087#diff-1b0d1f89ad78dd0ffc75e1b8a3a998c4bd7889d2f2aa426898c53d761112439bR12).